core/health/api/src/main/java/org/opennms/core/health/api/DefaultPassiveHealthCheck.java

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2018-2021 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2021 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.core.health.api;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A health check that does no action by its own but relies on being informed about healthiness.
+ */
+public class DefaultPassiveHealthCheck implements CachingHealthCheck {
+
+    private static final Logger LOG = LoggerFactory.getLogger(DefaultPassiveHealthCheck.class);
+
+    private final String description;
+    private final List<String> tags;
+
+    private Response cachedResponse = null;
+    private Instant cachedResponseTimestamp = null;
+
+    public DefaultPassiveHealthCheck(
+            String description,
+            List<String> tags,
+            Status initialStatus
+    ) {
+        this.description = description;
+        this.tags = tags;
+        setResponse(new Response(initialStatus));
+    }
+
+    @Override
+    public synchronized void setResponse(Response response) {
+        LOG.debug("cache response - healthCheck: {}; status: {}; msg: {}", description, response.getStatus(), response.getMessage());
```

Comment:
nit: we usually start LOG statements with Capitals.

Reply:
done